### DISCUSSION TITLE
Fan Site on Hinode

### DISCUSSION THREAD

**Discussion by Ipstenu** [gethinode/hinode#1098] [Show and tell]
The boring blah blah backstory is that I was forced to make a massive change to the site for legal reasons. You don't have to care. I don't care, and I was involved. But the end result was I took a site born in 1996 (yes, 1996) that had moved from HTML to SHTML to LiveJournal to home-grown PHP to WordPress all the way down to Hugo!

https://jorjafox.net

It includes my own Bootstrap Gallery mod and a **lot a lot** of custom theming to make the custom news, transcripts, and recap pages!

* https://github.com/JorjaFox/website
* https://github.com/JorjaFox/mod-bs5-lightbox

Here's the coolest thing (to me at least) - It took me about 3 days to port the bare bones of the site over, 3 weeks to curate the blog posts, and I'm now rebuilding some media setups, but with my lightbox mod, it's super easy to use.

So there you are :) Another site on Hinode!

**Comment by markdumay** [comment]
Thanks for your sharing experience @Ipstenu! I counted more than 1,500 pages on your fan site, so I'm impressed you were able to migrate the entire site in just a few weeks. Take care!

**Reply by Ipstenu** [reply]
1691 pages -- Batch jobs for the win :) I hand built a script that extracted, cleaned up, and then I just went through and spot-checked. There are still some design tweaks I want to make, but isn't that an ongoing life process?
